scripts/west_commands/zcmake.py

```diff
@@ -45,6 +45,7 @@ def run_cmake(args, cwd=None, capture_output=False, dry_run=False):
     _ensure_min_version(cmake, dry_run)
 
     cmd = [cmake] + args
+
```

Comment:
Why this blank line?

Reply:
That is probably due to moving code into this file, and then out again, as discussed:
https://github.com/zephyrproject-rtos/zephyr/pull/26060#discussion_r444777952

unfortunately this must have resulted in a blank line in that process.
You seem to be the first who noticed.